src/app.py

```diff
@@ -13,34 +15,179 @@
+                                    )
+                                ])
+                            ]),
+                            dbc.Row([
+                                html.Div([
+                                    html.H1(
+                                        "50",
+                                        style={'width': "150px", 'height': "150px", 'text-align': "center", 'vertical-align': "middle", 'color': "#DBA506"}
+                                    )
+                                ])
+                            ])
+                        ],
+                        width="auto")
+                    ]),
+                    # KPI Average Runtime, Average Rating
+                    dbc.Row([
+                        dbc.Col([
+                            dbc.Row([
+                                html.Div([
+                                    html.H6(
+                                        "Average Runtime",
+                                        style={'width': "150px", 'color': "#000000", 'font-weight': "bold", 'background': "#DBA506"}
+                                    )
+                                ])
+                            ]),
+                            dbc.Row([
+                                html.Div([
+                                    html.H1(
+                                        "120",
+                                        style={'width': "150px", 'height': "150px", 'text-align': "center", 'vertical-align': "middle", 'color': "#DBA506"}
+                                    )
+                                ])
+                            ])  
+                        ],
+                        width="auto"),
+                        dbc.Col([
+                            dbc.Row([
+                                html.Div([
+                                    html.H6(
+                                        "Average Rating",
+                                        style={'width': "150px", 'color': "#000000", 'font-weight': "bold", 'background': "#DBA506"}
+                                    )
+                                ])
+                            ]),
+                            dbc.Row([
+                                html.Div([
+                                    html.H1(
+                                        "10",
+                                        style={'width': "150px", 'height': "150px", 'text-align': "center", 'vertical-align': "middle", 'color': "#DBA506"}
+                                    )
+                                ])
+                            ])  
+                        ],
+                        width="auto")
+                    ]),    
+                ]),
+                dbc.Col([
+                    # Average Revenue/Runtime by Genre over time
+                    dbc.Row([
+                        html.Div([
+                            html.H6(
+                                "Average Revenue by Genre over time",
```

Comment:
```suggestion
                                "Average Rating by Genre over time",
```

Reply:
I suggest Rating as that is the default chart but like you said in Slack this will need to be made dynamic